python-threatexchange/threatexchange/cli/dataset/simple_serialization.py

```diff
@@ -87,3 +90,87 @@ def load(
                         )
                     )
         return ret
+
+
+class HMASerialization(CliIndicatorSerialization):
+    """
+    A Serialization for HMA Similar to CliIndicatorSerialization but with
```

Comment:
nit: add note to doc string that this serialization currently stores the "first descriptor" id as well as indicator id, future refactors should consider either removing this id or adding all visible descriptor ids.